var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -85,6 +85,8 @@ class Openmpi(AutotoolsPackage):
     patch('configure.patch', when="@1.10.0:1.10.1")
     patch('fix_multidef_pmi_class.patch', when="@2.0.0:2.0.1")
 
+    variant('rdma', default=False, description='Build with rdma-core; also need to manually set "fabrics" to include "verbs"')
```

Comment:
Would it be worth to just add a new value to mv variant `fabrics`? In this way you'll avoid the check at run-time, and you won't add a new variant (i.e. you'll be backward compatible with respect to changing hashes).